pallets/funding/src/lib.rs

```diff
@@ -18,96 +18,51 @@
 //!
 //! Polimec's main business logic. It allows users to create, evaluate, and fund projects.
 //!
-//! It rewards project evaluators and contributors with `Contribution Tokens`. These tokens
-//! can be redeemed for a project's native tokens, after their parachain is deployed on mainnet.
-//! ## 👷 Work in progress 🏗️
-//! Expect major changes between PRs
+//! It rewards project participants with Contribution Tokens. These tokens
+//! can be redeemed for a project's native tokens once the project goes live.
 //!
 //! ## Overview
-//! The official logic for Polimec's blockchain can be found at our [whitepaper](https://polimec.link/whitepaper).
+//! The official logic for Polimec's blockchain can be found at our [knowledge hub](https://hub.polimec.org/).
 //!
 //! There are 3 types of users in Polimec:
 //! - **Issuers**: They create projects and are responsible for their success.
-//! - **Evaluators**: They evaluate projects and are rewarded for their work.
-//! - **Contributors**: They contribute financially to projects and are rewarded on the basis of their contribution
+//! - **Evaluators**: They evaluate projects by locking PLMC and give them access to funding rounds. They are rewarded if the project gets funded, and slashed otherwise.
+//! - **Contributors**: They contribute financially to projects by locking PLMC and paying out USDT/USDC/DOT, and are rewarded in Contribution Tokens.
 //!
-//! A contributor, depending on their investor profile, can participate in different rounds of a project's funding.
+//! A user, depending on their investor profile, can participate in different ways
 //!
-//! There are 3 types of contributors:
-//! - **Institutional**
-//! - **Professional**
-//! - **Retail**
+//! There are 3 types of investors, which are authenticated by a KYC/AML process and a credential minted on the KILT blockchain:
+//! - **Institutional**: has access to creating projects, bidding, and contributing.
```

Comment:
has access to project creation, bidding, and contribution 

(please be aware to remove the dot)